src/pybind/mgr/devicehealth/module.py

```diff
@@ -118,6 +130,12 @@ def handle_command(self, _, cmd):
             return self.show_device_metrics(cmd['devid'], cmd.get('sample'))
         elif cmd['prefix'] == 'device check-health':
             return self.check_health()
+        elif cmd['prefix'] == 'device monitoring on':
+            self.set_config('enable_monitoring', 'true')
```

Comment:
Maybe kick self.event here?  That way if there is monitoring data that should raise health alerts, it will do so promptly rather than waiting for the next scrape.